client/src/components/Doctor/DoctorInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import { NavLink } from 'react-router-dom';
+import Avatar from '@mui/material/Avatar';
+import { Box } from '@mui/material';
+import IUser from '../../Interfaces/post/IUser';
+
+const UserPostInfo = ({ user }:{user:IUser|null|undefined}) => (
+  <NavLink to={`/users/${user?.id}`} className="user-info-post">
+    <Box sx={{ display: 'flex' }}>
+      <Avatar sx={{ width: '60px', height: '60px' }} alt={user?.name} src={user?.avatar} />
+      <Box sx={{
+        display: 'flex', alignItems: 'flex-start', flexDirection: 'column', justifyContent: 'space-around',
+      }}
+      >
```

Comment:
if youre using Box 
you can do:
`<Box display="flex" ....>`
SX is not needed. and should be used very rarely, as it has bad performance